src/services/codefixes/fixSpelling.ts

```diff
@@ -92,4 +96,8 @@ namespace ts.codefix {
 
         return context.program.getSourceFile(resolvedModule.resolvedFileName);
     }
+
+    function isPrivatePropertyAccessLike(name: string): boolean {
+        return name.charCodeAt(0) === CharacterCodes.hash && name.length > 1;

```

Comment:
You can either 
1. Look at getSuggestionForNonExistentSymbol and make a version that distinguishes between private identifiers and normal ones in some way. That's probably the best way, and the infrastructure is probably mostly in place. I'll take a look at how to do that when I get to the office.
2. `getPropertyOfType(getTypeOfSymbol(getSymbolOfNode((node.parent as PropertyAccessExpression).expression)), name)` and then check whether the property is private. I can't remember the best way to do that; you might check whether .valueDeclaration.kind is SyntaxKind.PrivateIdentifier.

Reply:
@sandersn Thanks. I checked the `getPropertyOfType` and seems it doesn't handle private identifiers., I found `getPrivateIdentifierPropertyOfType` which I applied instead.